app/src/androidTestKiwix/java/org/kiwix/kiwixmobile/utils/StandardActions.java

```diff
@@ -29,18 +36,22 @@
 public class StandardActions {
 
   public static void enterHelp() {
-    openContextualActionModeOverflowMenu();
-
-    onView(withText("Help"))
-        .perform(click());
+    BaristaSleepInteractions.sleep(250);
+    BaristaMenuClickInteractions.clickMenu(getResourceString(R.string.menu_help));
   }
 
   public static void enterSettings() {
-    openContextualActionModeOverflowMenu();
-
-    onView(withText("Settings"))
-        .perform(click());
+    BaristaSleepInteractions.sleep(250);
+    BaristaMenuClickInteractions.clickMenu(getResourceString(R.string.menu_settings));
   }
 
+  public static void deleteZimIfExists(String zimName, Integer adapterId) {
+    try {
+      onData(withContent(zimName)).inAdapterView(withId(adapterId)).perform(longClick());
+      clickDialogPositiveButton();
+    } catch (RuntimeException e) {
```

Comment:
As this is a helper method and called in various contexts, let's log details of whether the zim was found and whether we believe it's been deleted or whether the delete failed. I realise the original code didn't say much either, this is a good time to improve the information that's reported so future test runs can be analysed and problems addressed.